backend/src/main/java/com/happy/friendogly/playground/controller/PlaygroundController.java

```diff
@@ -0,0 +1,134 @@
+            "초코",
+            "뛰어놀기 좋아해요",
+            LocalDate.of(2024, 9, 27),
+            SizeType.LARGE,
+            Gender.FEMALE,
+            "https://i.pinimg.com/564x/d1/62/cb/d162cb12dfa0011a7bd67188a14d661c.jpg"
+    );
+    public static Pet dummyPet2 = new Pet(
+            new Member("김도선", "tag1", "imageUrl"),
+            "치키",
+            "가만히 있길 좋아해요",
+            LocalDate.of(2024, 9, 25),
+            SizeType.MEDIUM,
+            Gender.FEMALE,
+            "https://i.pinimg.com/564x/96/d4/43/96d443c92059f2b3a240a7ff74692bbf.jpg"
+    );
+
+    public static Pet dummyPet3 = new Pet(
+            new Member("박예찬", "tag2", "imageUrl"),
+            "토리",
+            "먹기 좋아해요",
+            LocalDate.of(2024, 9, 26),
+            SizeType.SMALL,
+            Gender.FEMALE,
+            "https://i.pinimg.com/564x/08/01/67/080167a359545bc40068045f984a9994.jpg"
+    );
+
+    @PostMapping
+    public ResponseEntity<ApiResponse<SavePlaygroundResponse>> save(
+            @Auth Long memberId,
+            @Valid @RequestBody SavePlaygroundRequest request
+    ) {
+        SavePlaygroundResponse savePlaygroundResponse = new SavePlaygroundResponse(
+                1L,
+                request.latitude(),
+                request.longitude()
+        );
+        return ResponseEntity.created(URI.create("/playgrounds/" + 1))
+                .body(ApiResponse.ofSuccess(savePlaygroundResponse));
+    }
+
+    @GetMapping("/{id}")
+    public ApiResponse<PlaygroundDetailResponse> find(@PathVariable Long id) {
+        return ApiResponse.ofSuccess(new PlaygroundDetailResponse(
+                        1L,
+                        20,
+                        10,
+                        List.of(
+                                PlaygroundPetDetail.dummyOf(
+                                        1L,
+                                        dummyPet1,
+                                        null,
+                                        true,
+                                        true
+                                ),
+                                PlaygroundPetDetail.dummyOf(
+                                        1L,
+                                        dummyPet2,
+                                        null,
+                                        true,
+                                        true
+                                ),
+                                PlaygroundPetDetail.dummyOf(
+                                        2L,
+                                        dummyPet3,
+                                        "7시에 가요~",
+                                        false,
+                                        false
+                                )
+                        )
+                )
+        );
+    }
+
+    @GetMapping("/locations")
+    public ApiResponse<List<PlaygroundLocationResponse>> findAllLocation() {
+        return ApiResponse.ofSuccess(
+                List.of(
```

Comment:
```suggestion
    @GetMapping("/locations")
    public ApiResponse<List<PlaygroundLocationResponse>> findAll() {
        return ApiResponse.ofSuccess(
                List.of(
```

Playground의 메인 정보가 어차피 Location이기 때문에 Location이라는 말은 없어도 될 것 같아요!

Reply:
위의 답변과 동일합니다!